app/assets/stylesheets/_sponsor_application.scss

```diff
@@ -78,3 +78,75 @@
 .steps.active:after {
   background: rgba(0, 0, 0, 0.5);
 }
+
+.status{
+  margin: 20px 0;
+  clear: both;
+}
+
+.status__heading{
+  line-height: 25px;
+  margin: 0;
+  margin-left: 10px;
+  float: left;
+  margin-bottom: 5px;
+  font-weight: normal;
+}
+
+.status__body{
```

Comment:
Selector `status__body` should be written in lowercase with hyphens<br>Opening curly brace `{` should be preceded by one space